vendor/assets/stylesheets/rolodex/components/_forms.sass

```diff
@@ -31,7 +31,8 @@ input[type="search"]
   box-sizing: border-box
 
 input[type="search"]::-webkit-search-cancel-button,
-input[type="search"]::-webkit-search-decoration
+input[type="search"]::-webkit-search-decoration,
+input[type="submit"]
   -webkit-appearance: none
```

Comment:
Since the selector isn't specific to Webkit I'd think we'd want all of the prefixes. We can use the Autoprefixer settings to update which prefixes are shown overtime if we'd like.